src/Line.php

```diff
@@ -56,6 +56,10 @@ class Line
 
     /**
      * @var string The input string which is assigned from the line parser. This is the actual content of the line itself!
+     * 
+     * @deprecated use getInput() instead
```

Comment:
Deprecation does not need a since tag, we should annotate deprecation messages as followed.

```php
/*
 * @deprecated Deprecated since [VERSION] will be removed in [REMOVE VERSION] use [REPLACE FN] 
 * instead.
 */
```